src/codeu/chat/server/Model.java

```diff
@@ -53,6 +56,9 @@ public int compare(Time a, Time b) {
 
   private static final Comparator<String> STRING_COMPARE = String.CASE_INSENSITIVE_ORDER;
 
+  private int collectiveSize = 0;
```

Comment:
Is this int just used to track the size of `transactionLog`? If so, remove it and just use the Queue's `size()` method.